internal/ci/ci.go

```diff
@@ -29,9 +28,9 @@ var (
 	// AWS represents https://aws.amazon.com/codebuild/
 	AWS = Provider{Name: "AWS CodeBuild", Envar: "CODEBUILD_INITIATOR"}
 	// Azure represents https://azure.microsoft.com/en-us/services/devops/
-	Azure = Provider{Name: "Azure DevOps", Envar: "Agent_BuildDirectory"}
+	Azure = Provider{Name: "Azure DevOps", Envar: "AGENT_ID"}
```

Comment:
That sounds like a very generic, perhaps too generic environment variable name. Could risk misclassifying the environment. Perhaps `TF_BUILD` is more suitable. I couldn't find many other hits for this name across the web, which would hopefully make it very unique to Azure.

Reply:
Nice！